cw/cw305/simple_capture_traces.py

```diff
@@ -179,13 +180,60 @@ def capture_kmac(ot, ktp):
         yield ret
 
 
+def capture_kmac_key(ot, ktp):
+    """A generator for capturing KMAC traces.
+    In order to simplify the analysis, the first sample has to use fixed key.
+
+    Args:
+      ot: Initialized OpenTitan target.
+      ktp: Key and plaintext generator.
+    """
+    key, _ = ktp.next()
```

Comment:
This key is never used as is the generator. So it could just be dropped. Alternatively, this generator could be more widely used instead of the fixed keys and texts + AES below. But then it wouldn't follow anymore the DTR spec.